<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-M19LQA3N" failures="1" tests="5" name="Submit Order Test" time="33.427" errors="0" timestamp="2024-11-29T19:53:23 IST">
  <testcase classname="abhiacademy.tests.SubmitOrderTest" name="submitOrder" time="7.9"/>
  <testcase classname="abhiacademy.tests.SubmitOrderTest" name="submitOrder" time="7.388"/>
  <testcase classname="abhiacademy.tests.SubmitOrderTest" name="validateOrder" time="1.02">
    <skipped/>
  </testcase> <!-- validateOrder -->
  <testcase classname="abhiacademy.tests.SubmitOrderTest" name="validateOrder" time="0.869">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at abhiacademy.pageobjects.MyOrderPage.checkOrderHisAndDel(MyOrderPage.java:48)
at abhiacademy.tests.SubmitOrderTest.validateOrder(SubmitOrderTest.java:62)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 19 stack frames]]>
    </failure>
  </testcase> <!-- validateOrder -->
  <testcase classname="abhiacademy.tests.SubmitOrderTest" name="validateOrder" time="1.293"/>
</testsuite> <!-- Submit Order Test -->
